src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyle = {};
+  const arrStr = sourceString.split(';');
+
+  for (const pieceArr of arrStr) {
```

Comment:
- arr/array/str/string/el/elem/obj/object/res/result are bad names, name should describe variable
- The task requires using iteration methods, the best option is `reduce` in this case, you generate an array with it